examples/connected-car-cluster-sharding/README.md

```diff
@@ -0,0 +1,141 @@
+## Akka Cluster Sharding Cloudflow Application
+
+### Problem Definition
+
+This project clusters a set of Akka Streamlets to demonstrate how leverage
+Akka Cluster Sharding for stateful stream processing in Cloudflow
+
+![](akka-cluster-streams.png)
+
+### Sub projects
+
+This application consists of the following sub-projects:
+
+* `akka-connected-car`: Contains the blueprint
+* `akka-connected-car-streamlet`: Akka Streams based ingress, cluster streamlet, and logger
+* `datamodel`: Contains the Avro schema
+
+### Example Deployment example on GKE
+
+**Steps:**
+
+* Make sure you have installed a GKE cluster with Cloudflow running as per the [installation guide](https://github.com/lightbend/cloudflow-installer).
+Make sure you have access to your cluster:
+
+```
+gcloud container clusters get-credentials <CLUSTER_NAME>
+```
+
+and that you have access to the Google docker registry:
+
+```
+gcloud auth configure-docker
+```
+
+* Add the Google docker registry to your sbt project (should be adjusted to your setup). The following lines should be there in the file `target-env.sbt` at the root of your application. e.g.
+
+```
+ThisBuild / cloudflowDockerRegistry := Some("eu.gcr.io")
+ThisBuild / cloudflowDockerRepository := Some("my-awesome-project")
+```
+
+`my-awesome-project` refers to the project ID of your Google Cloud Platform project.
+
+* Build the application:
+
+```
+$ sbt buildAndPublish
+```
+
+At the very end you should see the application image built and instructions for how to deploy it:
+
+```
+[info] 221-65c1693-dirty: digest: sha256:3377f15b53ad46781dd2396c077dff2c78d8f4ccd102d272ce97d5af7b8fcd86 size: 6804
+[info]
+[info] Successfully built and published the following Cloudflow application image:
+[info]
+[info]   eu.gcr.io/my-awesome-project/connected-car-akka-cluster:221-65c1693-dirty
+[info]
+[info] You can deploy the application to a Kubernetes cluster using any of the the following commands:
+[info]
+[info]   kubectl cloudflow deploy eu.gcr.io/my-awesome-project/connected-car-akka-cluster:221-65c1693-dirty
+[info]
+[success] Total time: 317 s (05:17), completed Apr 14, 2020 10:30:03 PM
+```
+
+* Make sure you have the `kubectl cloudflow` plugin configured.
+
+```
+$ kubectl cloudflow help
+This command line tool can be used to deploy and operate Cloudflow applications.
+...
+```
+
+* Deploy the app.
+
+```
+$ kubectl cloudflow deploy -u oauth2accesstoken eu.gcr.io/my-awesome-project/connected-car-akka-cluster:221-65c1693-dirty -p $(gcloud auth print-access-token)"
+Existing value will be used for configuration parameter 'cdr-generator2.records-per-second'
```

Comment:
Is this output from call record aggregator by mistake?

Reply:
Good catch!  sorry forgot to replace this section